### ISSUE TITLE
BUG: Repair

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#1690]
![image](https://user-images.githubusercontent.com/47291794/187032293-4cfea3dc-3f95-4fde-919a-daed3e6a634e.png)
1. Repair cost at 90-100% is 5 gold.
2. Repair cost at 80-100% is 10 gold.
3. Repair cost at 0-100% is 45 gold.
4. In the video, we see that up to 90% the cost is 5 gold. This is right. But already at 89%, we see that the repair costs 45 gold, and it should have been 10 gold. That is, after 89% and below, the repair costs as if the item was completely broken.

After the first element, the last element is used instead of the second.

https://youtu.be/aL72v0rM_g8

**Comment by insthync** [comment]
In `BaseItem_Repair`

Changes loop codes in function: `GetRepairPrice` to

```
for (int i = itemRefine.RepairPrices.Length - 1; i >= 0; --i)
{
...
}
```

I will add sort codes to make sure it will be looped correctly for all user's settings.

**Comment by insthync** [comment]
Fixed in 1.77
